src/main/java/christmas/domain/Badge.java

```diff
@@ -0,0 +1,24 @@
+package christmas.domain;
+
+public enum Badge {
+    SANTA("산타", 20000),
+    TREE("트리", 10000),
+    STAR("별", 5000);
+
+    private final String name;
+    private final int discountAmount;
+
+    Badge(String name, int discountAmount) {
+        this.name = name;
+        this.discountAmount = discountAmount;
+    }
+
+    public static String calculateBadge(int discountAmount) {
+        for (Badge badge : Badge.values()) {
+            if (-discountAmount >= badge.discountAmount) {
+                return badge.name;
+            }
+        }
+        return null;
```

Comment:
```suggestion
        return Arrays.stream(Badge.values()).filter(badge -> badge.discountAmount <= - discountAmonut).findAny().orElse(null)
    }
```

으로 고쳐볼 수 있을 것같네요!
(enum에 대해 리뷰달아주셔서 저도 한스푼 ㅎㅎ)

Reply:
제가 할인 금액을 음수값을 상수로 사용해서 양수로 만들기 위해서 -를 붙이게 되었습니다 ㅎㅎ..